### DISCUSSION TITLE
Tren Finance 需要 Referral code

### DISCUSSION THREAD

**Discussion by linghuccc** [IntensiveCoLearning/DeFi#24] [Q&A] [answered]
不然什么都看不到
![error](https://github.com/user-attachments/assets/841928d1-7394-4196-b2fc-4176155f88a6)

**Comment by mon1y** [comment, marked as answer]
Z8B0F 这是我的邀请码 大家使用后 可以把邀请码发出来给下一个人用

**Reply by linghuccc** [reply]
上面的邀请码，我已经用了。

D1U5C 是我的邀请码，请用这个邀请码的人，回复这个帖子，谢谢。

**Reply by jjeejj** [reply]
D1U5C 已经使用 --> M5I4B 新的

**Reply by muxin-web3** [reply]
M5I4B 已使用 --> R1K2E 新的

**Reply by EgoSay** [reply]
R1K2E 已使用 --> A3R4I 新的

**Reply by luffythink** [reply]
A3R4I 已使用 -->  新的M2C4N

**Reply by DerickIT** [reply]
A3R4I 已使用 --> 新的G1Q2X

**Reply by punkcanyang** [reply]
G1Q2X 已使用 --> 新的L2Q4C

**Reply by YuKirasawa** [reply]
L2Q4C 已使用 --> 新的 K5N0F

**Reply by joyc** [reply]
K5N0F 已使用 --> 新的 H1P9X

**Comment by zedz** [comment]
H1P9X 已使用 --> 新的 D8D2I

**Reply by wayhome** [reply]
D8D2I 已使用 --> 新的 D0O8E

**Reply by Huowuge** [reply]
I0Z4I

**Reply by rayjun** [reply]
K3C7U

**Reply by L1rul1n** [reply]
K3C7U 已使用 --> 新的 G8B4D

**Reply by TheHeBoy** [reply]
R3C8P

**Reply by nocb** [reply]
R3C8P  我已经用了， 但是我不知道怎么生成新的

**Reply by nocb** [reply]
R3C8P ---》 R8T7Y

**Reply by novac42code** [reply]
> R3C8P ---》 R8T7Y

已使用，新的U0M5W

**Reply by boombb12138** [reply]
U0M5W---》 D3S3V
